packages/devtools_app/lib/src/framework/notifications_view.dart

```diff
@@ -267,6 +269,29 @@ class _NotificationState extends State<_Notification>
   }
 }
 
+class _DismissAction extends StatelessWidget {
+  const _DismissAction({
+    required this.widget,
```

Comment:
instead of passing in a widget, can we just pass in an `onPressed` callback?

Reply:
Done!